xbmc/windowing/gbm/OffScreenModeSetting.cpp

```diff
@@ -0,0 +1,43 @@
+#include "OffScreenModeSetting.h"
+#include "utils/log.h"
+
+bool COffScreenModeSetting::InitDrm()
+{
+  if (!CDRMUtils::OpenDrm(false))
+  {
+    return false;
+  }
+
+  CLog::Log(LOGDEBUG, "COffScreenModeSetting::%s - initialized offscreen DRM", __FUNCTION__);
+  return true;
+}
+
+void COffScreenModeSetting::DestroyDrm()
+{
+  close(m_fd);
+  m_fd = -1;
+}
+
+std::vector<RESOLUTION_INFO> COffScreenModeSetting::GetModes()
+{
+    std::vector<RESOLUTION_INFO> resolutions;
+    resolutions.push_back(GetCurrentMode());
+    return resolutions;
+}
+
+RESOLUTION_INFO COffScreenModeSetting::GetCurrentMode()
+{
+  RESOLUTION_INFO res;
+  res.iScreen = 0;
```

Comment:
is already gone is master

Reply:
Oh yeah thx - I rebased on one machine transferred to the other and only built x11...